test/OWNERS

```diff
@@ -0,0 +1,10 @@
+# The OWNERS file is used by prow to automatically merge approved PRs.
+
+approvers:
+- productivity-approvers
+
+reviewers:
+- productivity-reviewers
+
+labels:
+- area/test-and-release
```

Comment:
in order for this to work you need to add `productivity-approvers` and `productivity-reviewers` to `./OWNER_ALIASES`. See example here: https://github.com/knative/eventing/blob/master/OWNERS_ALIASES